python/triton/ops/block_reduce.py

```diff
@@ -0,0 +1,56 @@
+import torch
+import triton.language as tl
+import triton
+
+'''
+[split_k, m, n] -> [m, n]
```

Comment:
Add a bit more comment here on what's the reduction doing? Is it hardcoded to do a sum() right now below? perhaps it can be other types of operations like min/max etc.?